frontend/.vscode/settings.json

```diff
@@ -33,4 +33,5 @@
     },
     "editor.formatOnSave": true,
     "files.eol": "\n",
+    "cmake.configureOnOpen": false,
```

Comment:
Only include changes relevant to the issue

Reply:
okay